src/components/Calculator.js

```diff
@@ -1,30 +1,10 @@
-import './Calculator.css';
+import React from 'react'
+import CalcDisplay from './CalcDisplay'
 
-const Calculator = () => (
-  <div className="container">
-    <div className="result-display">0</div>
-    <div className="calc-buttons">
-      <button className="calc-button" type="button">AC</button>
-      <button className="calc-button" type="button"> +/- </button>
-      <button className="calc-button" type="button"> % </button>
-      <button className="calc-button-right" type="button"> ÷ </button>
-      <button className="calc-button" type="button"> 7 </button>
-      <button className="calc-button" type="button"> 8 </button>
-      <button className="calc-button" type="button"> 9 </button>
-      <button className="calc-button-right" type="button"> x </button>
-      <button className="calc-button" type="button"> 4 </button>
-      <button className="calc-button" type="button"> 5 </button>
-      <button className="calc-button" type="button"> 6 </button>
-      <button className="calc-button-right" type="button"> - </button>
-      <button className="calc-button" type="button"> 1 </button>
-      <button className="calc-button" type="button"> 2 </button>
-      <button className="calc-button" type="button"> 3 </button>
-      <button className="calc-button-right" type="button"> + </button>
-      <button className="calc-button" id="zero" type="button"> 0 </button>
-      <button className="calc-button" type="button"> . </button>
-      <button className="calc-button-right" type="button"> = </button>
-    </div>
-  </div>
-);
+const Calculator = () => {
+  return (
+    <CalcDisplay></CalcDisplay>
```

Comment:
- [ ] [Optional] You can self-close this component. Here's how: `<CalcDisplay />`

Reply:
@jaonfelice Thanks for the review 💯 . Yeah, it was automatically done while fixing linters.

https://github.com/ngeelani48/math-magicians/pull/2/commits/a43f205ac347fbd2c2940e03fd0c8532ff4adcd5#diff-2e862d0b88bdd42eb13f6528bc2ef48a56500daca0350696b337c7f2127f4f0bR4-R6